CONTRIBUTING.md

```diff
@@ -5,12 +5,13 @@ Contribution to this project is very much like contributing to any other Free/Li
 3. Make a new branch from `develop` branch (This is important here! You need to checkout to `develop` branch and then make a branch)
 4. Make the local development environment ready by running `npm install`.
 5. To run the project on live server, run `npm run watch`.
-6. Commit your changes.
-7. Push your commits to your own fork
-8. Send Pull Request from your new branch to this project's **develop branch**.
+6. To run the project on local host, run `npm run build` then `npm start`.
+7. Commit your changes.
+8. Push your commits to your own fork
+9. Send Pull Request from your new branch to this project's **develop branch**.
 
 You are done. I will get notified and then take the next steps accordingly.
 
 If later you need to contribute, you need to first make sure your fork is up to date. You should see a button on your fork for synchronization. After that, you need to use `git pull` on your clone (on you local machine) to get latest commits and then the rest is the same...
 
-This project is under AGPL license. 
+This project is under AGPL license.
```

Comment:
Please send this as a separate PR

Reply:
Hello @ahangarha ,
I have made the requested changes to this pull request. Thank you.

Reply:
Still this line shouldn't be included in this PR. This is not related to your own project. Please revert this change in this PR

Reply:
I decided to ignore this little irrelevant issue and proceed for merge.

Please keep in mind to keep your commits in PR all relevant to the PR topic.

Thanks again.